ports/xed/portfile.cmake

```diff
@@ -11,8 +11,8 @@ vcpkg_from_github(
 vcpkg_from_github(
     OUT_SOURCE_PATH MBUILD_SOURCE_PATH
     REPO intelxed/mbuild
-    REF f01087a20b6ac4bd99f7a7018ac467c2d54d3d2b
-    SHA512 1f2ea4f1622907bd4aa63329b1b557a75f10dd68b7d22cbb2f4b674e50a7fc643fe2e4e9d0101c0d7dbae280ad559c2d4a3f7f84cf2f491efeffec77f823c3d2
+    REF 3e8eb33aada4153c21c4261b35e5f51f6e2019e8
```

Comment:
I bumped this just far enough to include [this fix](https://github.com/intelxed/mbuild/commit/3e8eb33aada4153c21c4261b35e5f51f6e2019e8).

I could bring it further up to date. Or I could even add a patch under `ports/` like we do with LLVM. Not sure what is preferable.